css/style.css

```diff
@@ -134,6 +139,15 @@ footer p:last-child {
   margin-bottom: 0;
 }
 
+.nav {
+  margin-bottom: 0;
+  padding: 20px auto;
```

Comment:
For me, Firefox gives an "Invalid property value" error, because `padding` can't have an `auto` value.

```suggestion
```

Removing it also works for me.

Reply:
Sure.